.github/workflows/upload-docs.yml

```diff
@@ -0,0 +1,80 @@
+name: Build and upload API docs
+
+on:
+  release:
+    types: [published]
+
+env:
+  GH_TOKEN: ${{ github.token }}
+
+permissions:
+  actions: 'write'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Get release language
+        id: get_release_language
+        run: |
+          if [[ ${{ github.ref }} == *"python"* ]]; then
+            echo LANGUAGE="python" >> $GITHUB_OUTPUT
+          fi
+          if [[ ${{ github.ref }} == *"nodejs"* ]]; then
+            echo LANGUAGE="nodejs" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Check if release language is supported
+        if: steps.get_release_language.outputs.LANGUAGE != 'python' && steps.get_release_language.outputs.LANGUAGE != 'nodejs'
+        run: gh run cancel ${{ github.run_id }}
+
+      - name: Set up Rust
+        uses: ./.github/actions/setup-rust
+
+      - name: Set Up Node.js 18 and Yarn Cache
+        uses: actions/setup-node@v3
+        with:
+          node-version: "18"
+
+      - name: Install Required Dependencies (Ubuntu)
+        run: |
+          sudo apt-get update
+          sudo apt-get install libudev-dev libusb-1.0-0-dev
+
+      - name: Get release version
+        id: get_release_version
+        run: |
+          VERSION=$(echo ${{ github.ref }} | sed -e 's/.*v\([0-9]*\.[0-9]*\).*/\1/')
```

Comment:
If not, I would put a TODO

Reply:
Sorry, had to remember again what I planned 😆 
It would always put all versions with the same major and minor in the same folder.
So it would put rc, alpha and so on in the same folder, override it.
So `v1.2-rc.1` would be pushed to the a folder called `1.2`. And once you release `1.2` it would override the `rc` docs, as at that point there is no need for the `rc` anymore and everyone should use the stable version. Makes sense? 😅

Reply:
Ok fair enough, thanks